sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubErrorCodeStrings.java

```diff
@@ -0,0 +1,287 @@
+
+    /**
+     * The value of {@code events} key is null
+     */
+    static final String EVENTS_CANNOT_NULL = "null_events";
+
+    /**
+     * The value of {@code eventData} key is null
+     */
+    static final String EVENT_DATA_CANNOT_NULL = "null_event_data";
+
+    /**
+     * The value of {@code eventPosition} key is null
+     */
+    static final String EVENT_POSITION_CANNOT_NULL = "null_event_position";
+
+    /**
+     * The value of {@code eventHubName} key is null
+     */
+    static final String EVENTHUB_NAME_CANNOT_NULL = "null_eventhub_name";
+
+    /**
+     * The value of {@code host} key is null
+     */
+    static final String HOST_CANNOT_NULL = "null_host";
+
+    /**
+     * The value of {@code initialEventPosition} key is null
+     */
+    static final String INITIAL_EVENT_POSITION_CANNOT_NULL = "null_initial_event_position";
+
+    /**
+     * The value of {@code message} key is null
+     */
+    static final String MESSAGE_CANNOT_NULL = "null_message";
+
+    /**
+     * The value of {@code options} key is null
+     */
+    static final String OPTIONS_CANNOT_NULL = "null_options";
+
+    /**
+     * The value of {@code partitionId} key is null
+     */
+    static final String PARTITION_ID_CANNOT_NULL = "null_partition_id";
+
+    /**
+     * The value of {@code partitionProcessorFactory} key is null
+     */
+    static final String PARTITION_PROCESSOR_FACTORY_CANNOT_NULL = "null_partition_processor_factory";
+
+    /**
+     * The value of {@code policyName} key is null
+     */
+    static final String POLICY_NAME_CANNOT_NULL = "null_policy_name";
+
+    /**
+     * The value of {@code producer} key is null
+     */
+    static final String PRODUCER_CANNOT_NULL = "null_producer";
+
+    /**
+     * The value of {@code key} key is null
+     */
+    static final String PROPERTY_KEY_CANNOT_NULL = "null_property_key";
+
+    /**
+     * The value of {@code value} key is null
+     */
+    static final String PROPERTY_VALUE_CANNOT_NULL = "null_property_value";
+
+    /**
+     * The value of {@code handlerProvider} key is null
+     */
+    static final String REACTOR_HANDLER_PROVIDER_CANNOT_NULL = "null_reactor_handler_provider";
+
+    /**
+     * The value of {@code provider} key is null
+     */
+    static final String REACTOR_PROVIDER_CANNOT_NULL = "null_reactor_provider";
+
+    /**
+     * The value of {@code sharedAccessKey} key is null
+     */
+    static final String SHARED_ACCESS_KEY_CANNOT_NULL = "null_shared_access_key";
+
+    /**
+     * The value of {@code tokenValidity} key is null
+     */
+    static final String TOKEN_VALIDITY_CANNOT_NULL = "null_token_validity";
+
+    /**
+     * The value of {@code tracerProvider} key is null
+     */
+    static final String TRACER_PROVIDER_CANNOT_NULL = "null_tracer_provider";
+
+    /**
+     * The value of {@code tryTimeout} key is null
+     */
+    static final String TRY_TIME_OUT_CANNOT_NULL = "null_try_time_out";
+
+
+    // EMPTY STRINGS
```

Comment:
What does this mean?

Reply:
Trying to add more description for the errors messages of "something can not be empty strings" . Removed this comment already